packages/openapi-v3/src/__tests__/unit/decorators/deprecated.decorator.unit.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {anOpenApiSpec} from '@loopback/openapi-spec-builder';
+import {expect} from '@loopback/testlab';
+import {getControllerSpec} from '../../..';
+import {api, get, oas} from '../../../decorators';
```

Comment:
Can we merge the import statement with the one above?

Reply:
Done!